api/v1beta1/imageupdateautomation_types.go

```diff
@@ -27,7 +27,8 @@ const ImageUpdateAutomationKind = "ImageUpdateAutomation"
 // ImageUpdateAutomationSpec defines the desired state of ImageUpdateAutomation
 type ImageUpdateAutomationSpec struct {
 	// SourceRef refers to the resource giving access details
-	// to a git repository.
+	// to a git repository. It must be in the same namespace as the
+	// ImageUpdateAutomation.
```

Comment:
It's not totally obvious what "it" is referring to. Perhaps put the constraint in line with the referent:

```go
// SourceRef refers to the GitRepository object in the same namespace that has access details.
```